examples/ActionGroup/ActionGroup.Tests.ps1

```diff
@@ -48,11 +87,12 @@ Describe 'Verify Action Group Exists with Custom Assertion' {
 
 Describe 'Verify Action Group Exists in Correct Location' {
   it 'Should contain an action group named sampleaction in global' {
-    $resourceGroupName = "rg-test"
-    $actionGroupName = "sampleaction"
+    #arrange
+    $rgName = $rgName
+    $actionGroupName = $resourceName
 
     #act
-    $result = Confirm-AzBPActionGroup -ResourceGroupName $resourceGroupName -ActionGroupName $actionGroupName
+    $result = Confirm-AzBPActionGroup -ResourceGroupName $rgName -ActionGroupName $actionGroupName
 
     #assert
     $result | Should -BeInLocation 'global'
```

Comment:
```suggestion
    $result | Should -BeInLocation $location
```
If we do go with script vars, might have to change these location values too so user can easily customize

Reply:
This might be a topic for a design session. There are a lot of factors that might be in play. Some locations are determined by the logged in user automatically when the bicep file is deployed, others you can set in the bicep file directly. A single source of location would be nice though.